TrafficCrash/Models/VehiclePeopleCrashDetails.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace TrafficCrash.Models
 {
-    public class Vehicle_People_Crash_details
+    public class VehiclePeopleCrashDetails
```

Comment:
This aligns better with widely accepted naming conventions.  We tend to avoid _ in class names, except for test classes.